apps/examples/elf/micomapp/Makefile

```diff
@@ -97,6 +97,14 @@ KERNEL_VER = 2.0
 STACKSIZE = 2048
 PRIORITY = 220
 
+ifeq ($(CONFIG_COMPRESSED_BINARY),y)
```

Comment:
If we get them from configs, do we need to add them here?
Without this, we can add them at mkbinheader.py directly. Then, we don't need to modify Makefile.
How about your opinion?

Reply:
ok..  will look into how to make this change.
It makes sense to do this.

Reply:
@sunghan-chang 
I am not able to find a way to get config values in mkbinheader python file.
I found one "ConfigParser" module but that expects config files to have section headers. Without section headers, it returns "MissingSectionHeaderError".

Please suggest how to read/parse config file in python.

Reply:
@amandeepc I don't know it either. Let's keep it of now before knowing.